### ISSUE TITLE
Just to notify you guys :)

### ISSUE THREAD

**Issue by lamebread** [swisstxt/streamplayer#5]
Hey there,

That tool is still running, http://streamplayer.swisstxt.ch/.

Just to let you know :)

I hope you're all well, have a nice day ;)

@onitake @joschi36 @gildegoma

Liebi Grüess,
Maël

**Comment by onitake** [comment]
It's open source. :wink: 

How's it going? Not getting bored at your new job?

**Comment by lamebread** [comment]
@onitake Yeah, just wanted to let you know that there's a random tool, written by an ex-apprentice which is running publicly under a subdomain ;D

It's going pretty well, yes. I switched job again, from DevOps Engineer to Linux System Engineer. Currently attending a Cisco Hackathon. 

How are you guys doing? I guess at the moment it's hard to meet each other physically, maybe we could organise a virtual FüBi with a couple of other ex-TXTlers and current TXTlers :)

**Comment by onitake** [comment]
Oh wow. Does that mean you also switched companies?

FüBi sounds good. I'll try to put something together with Nik.

**Comment by lamebread** [comment]
@onitake Sorry, I always forget to check out Github. We use GitLab, so I'm not on here a lot :D 

Yeah, I now work at SPIE in Bern. It's quite funny, the team structure and spirit is very similar to when I was at SwissTXT, so it feels kind of "back to the roots". But it's very refreshing to be working in the Linux space again.

Nice :) See ya sometime soon, be it virtually or phisically.

**Comment by lamebread** [comment]
By the way, if you go to http://streamplayer.swisstxt.ch/ and put "Daniel" in the text box, you can experience an easter-egg

**Comment by onitake** [comment]
Fabulous!

By the way, your website directly embeds content from Facebook, LinkedIn and Xing without asking. Haven't gotten any GDPR complaints yet? :stuck_out_tongue_winking_eye:

**Comment by lamebread** [comment]
That tool was developed in the good old times, when you didn't have to make your users aware that you're abusing their data.

But no, haven't heard anything about that ;D If I do, I'll forward the complaints to ***@swisstxt.ch

**Comment by onitake** [comment]
I meant your employer's website, not the streamplayer. :wink:

**Comment by lamebread** [comment]
Ah, you're right, never noticed that. I'll see what happens if I mention it :)

swisstxt.ch does not show me one, either :smirk: Let's see which company is faster to implement it ;D

**Comment by onitake** [comment]
Oh yeah, I might want to drop our legal team a note... CH is not in the EU, but we're not blocking outside access, so....
